cfgov/filing_instruction_guide/models.py

```diff
@@ -169,16 +69,47 @@ class FIGContentPage(CFGOVPage):
         ]
     )
 
+    def get_toc_headers(self, request):
+        toc_headers = []
+        parent = None
+        for section in self.content:
+            header = section.value.get("header")
+            id = section.value.get("section_id")
+            if section.block_type == "Fig_Section":
+                if parent:
+                    toc_headers.append(parent)
+                parent = {"header": header, "id": id, "children": []}
+            elif section.block_type == "Fig_Sub_Section":
+                # if the first block is a subsection instead of a section
+                if not parent:
+                    parent = {"header": "", "id": "", "children": []}
+                parent["children"].append({"header": header, "id": id})
+        toc_headers.append(parent)
+        return toc_headers
+
+    def assign_section_ids(self):
+        ind = sub_ind = sub3_ind = 0
+        for section in self.content:
+            id = ""
+            sec_type = section.block_type
+            if sec_type == "Fig_Section":
+                ind += 1
+                sub_ind = 0
+                sub3_ind = 0
+                id = f"{ind}."
```

Comment:
I think it would be better to remove the trailing period here (and on lines 103 and 106). I don't think the trailing periods add much clarity and we'd also hew more closely to https://www.consumerfinance.gov/rules-policy/regulations/ etc

Reply:
Interesting. We included the trailing period because it was included in the original design by @nataliafitzgerald (which you can see in GHE/super-regular/issues/73). Since they'd be easy to remove later, maybe we could consult a designer and make that improvement later?

Reply:
Thank you @wpears for reviewing!